.github/workflows/labels.yml

```diff
@@ -26,8 +24,12 @@ jobs:
       - name: install deps
         run: npm ci
 
-      - name: clear existing labels
-        run: npx ghlbl -o eddiejaoude -r stargate -t ${{ secrets.GITHUB_TOKEN }} -d
-
-      - name: import labels from json
-        run: npx ghlbl -o eddiejaoude -r stargate -t ${{ secrets.GITHUB_TOKEN }} -i docs/labels.json
+      - name: check label file for change
+        uses: pheel/path-watcher-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          paths: 'docs/labels.json'
+      - if: steps.modified.outputs.modified
+        run: |
+          npx ghlbl -o eddiejaoude -r stargate -t ${{ secrets.GITHUB_TOKEN }} -d
+          npx ghlbl -o eddiejaoude -r stargate -t ${{ secrets.GITHUB_TOKEN }} -i docs/labels.json
```

Comment:
Did the default `path` option in the YML settings not work? 🤔

Reply:
https://github.community/t/matching-branch-and-path-not-working/17011/2

This forum post suggests that it should work 🤔

Reply:
It will be `OR`

This config would trigger on `branch` or `path` rather than `AND` 😢 

```yml
on:
  push:
    branches:
      - eddiejaoude-issue-56
    path:
      - 'docs/labels.json'
```

Reply:
Hmm could flip it around and check for the path, then do the condition for the branch in the step 🤔 . No 3rd party plugin would be needed

Reply:
I went for that option, it looks cleaner and should trigger less, as the file will hardly change, but still only imports on once merged to `main`